tools/rtk_util.py

```diff
@@ -61,9 +61,8 @@ async def do_load(usb_transport, force):
         # Get the driver.
         driver = await rtk.Driver.for_host(host, force)
         if driver is None:
-            if not force:
```

Comment:
I think it's erroneous to only return if `not force` -- otherwise, it would then try to `driver.download_firmware()`, which would surely fail if `driver is None`.